src/main/resources/block_mappings.json

```diff
@@ -132,14 +132,14 @@
     "1": "minecraft:spruce_log;pillar_axis=y",
     "2": "minecraft:birch_log;pillar_axis=y",
     "3": "minecraft:jungle_log;pillar_axis=y",
-    "4": "minecraft:oak_log;pillar_axis=x",
-    "5": "minecraft:spruce_log;pillar_axis=x",
-    "6": "minecraft:birch_log;pillar_axis=x",
-    "7": "minecraft:jungle_log;pillar_axis=x",
-    "8": "minecraft:oak_log;pillar_axis=z",
-    "9": "minecraft:spruce_log;pillar_axis=z",
-    "10": "minecraft:birch_log;pillar_axis=z",
-    "11": "minecraft:jungle_log;pillar_axis=z"
+    "8": "minecraft:oak_log;pillar_axis=x",
+    "9": "minecraft:spruce_log;pillar_axis=x",
+    "10": "minecraft:birch_log;pillar_axis=x",
+    "11": "minecraft:jungle_log;pillar_axis=x",
+    "16": "minecraft:oak_log;pillar_axis=z",
+    "17": "minecraft:spruce_log;pillar_axis=z",
+    "18": "minecraft:birch_log;pillar_axis=z",
+    "19": "minecraft:jungle_log;pillar_axis=z"
```

Comment:
why does this not do the sequence of numbers correctly it's really weird minecraft you can't find